cardano-api/ChangeLog.md

```diff
@@ -5,6 +5,13 @@
 
 ### Features
 
+- New error exports:
```

Comment:
error exports?

Reply:
We need to be able to construct errors in order to test how they display.